overlord/hookstate/ctlcmd/refresh.go

```diff
@@ -96,6 +102,15 @@ func init() {
 	cmd.hidden = true
 }
 
+func (c *refreshCommand) nonRootExecute(context *hookstate.Context) error {
+	switch {
+	case c.Tracking:
+		return c.printTrackingInfo(context)
+	default:
+		return ForbiddenCommandError{Message: "non-root users can only use --tracking with the refresh command"}
```

Comment:
I believe this should be 
```suggestion
		return &ForbiddenCommandError{Message: "non-root users can only use --tracking with the refresh command"}
```

See daemon/api_snapctl.go for the place where this gets used